src/Microsoft.Crank.Agent/Startup.cs

```diff
@@ -1867,6 +1881,61 @@ private static async Task<(string containerId, string imageName, string workingD
                     {
                         job.Error = job.BuildLog.ToString();
                     }
+
+                    var dockerInspectArguments = $"inspect -f \"{{{{ .Size }}}}\" {imageName}";
```

Comment:
Does it actually work? At first I was wondering if the resulting unit was actually KB, but then it appeared it didn't work for me since it's an image, not a container, so you can't do inspect. What worked was `docker images golang --format "{{ .Size }}"` and the result contains the unit, like `862MB`

Reply:
For a container, the property was `{{ .SizeRw }}` in bytes

Reply:
Yes, it works. On my machine it produces the same output as `docker image inspect`. I think Docker is smart enough to figure out what you're asking for, so passing it an image name does the correct thing.

The raw value looks like this:
```
        "Size": 244714070,
```
Which lines up with the size from `docker image ls`:
```
benchmarks  latest   275ac07cf3fb   2 weeks ago     245MB
```

However, you're right that I'm actually logging bytes, not KB. I'll fix that.

full output:
<details>

```
❯ docker inspect  benchmarks
[
    {
        "Id": "sha256:275ac07cf3fb6d34d33dc6b72f21458d951fbffdb7e05a3f699835c8e17055f4",
        "RepoTags": [
            "benchmarks:latest"
        ],
        "RepoDigests": [],
        "Parent": "",
        "Comment": "buildkit.dockerfile.v0",
        "Created": "2021-04-24T20:19:10.5097035Z",
        "Container": "",
        "ContainerConfig": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": null,
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "",
            "Entrypoint": null,
            "OnBuild": null,
            "Labels": null
        },
        "DockerVersion": "",
        "Author": "",
        "Config": {
            "Hostname": "",
            "Domainname": "",
            "User": "",
            "AttachStdin": false,
            "AttachStdout": false,
            "AttachStderr": false,
            "Tty": false,
            "OpenStdin": false,
            "StdinOnce": false,
            "Env": [
                "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
                "ASPNETCORE_URLS=http://+:80",
                "DOTNET_RUNNING_IN_CONTAINER=true",
                "DOTNET_VERSION=6.0.0-preview.4.21221.4",
                "ASPNET_VERSION=6.0.0-preview.4.21221.10",
                "Logging__Console__FormatterName=Json"
            ],
            "Cmd": null,
            "Image": "",
            "Volumes": null,
            "WorkingDir": "/app",
            "Entrypoint": [
                "dotnet",
                "Benchmarks.dll"
            ],
            "OnBuild": null,
            "Labels": null
        },
        "Architecture": "amd64",
        "Os": "linux",
        "Size": 244714070,
        "VirtualSize": 244714070,
        "GraphDriver": {
            "Data": {
                "LowerDir": "/var/lib/docker/overlay2/qn271qirrqnz8ydzs72xzjoxx/diff:/var/lib/docker/overlay2/4f5a007e0140f2504866786c255afed9aec0f53e36d91225c838f5e4b6d0c1dc/diff:/var/lib/docker/overlay2/09160f6171461ccd54515f8001613a31e6e53f4e6908460413d2fd336d34670e/diff:/var/lib/docker/overlay2/afdadcc2986c542ee596d4309bf197c3589210d6a0d62327ee19fd385d2a6911/diff:/var/lib/docker/overlay2/65244f19344582ebcd707a4b1d00ec2f91ad8550026b9a66bb0918c647079d16/diff:/var/lib/docker/overlay2/3b253c4070a8ee589d32d942ef68a59ace4ed612d6cf3a8d5b110b3b1bc5d824/diff",
                "MergedDir": "/var/lib/docker/overlay2/xdcfdtkea3b4uja9c86nwzs5c/merged",
                "UpperDir": "/var/lib/docker/overlay2/xdcfdtkea3b4uja9c86nwzs5c/diff",
                "WorkDir": "/var/lib/docker/overlay2/xdcfdtkea3b4uja9c86nwzs5c/work"
            },
            "Name": "overlay2"
        },
        "RootFS": {
            "Type": "layers",
            "Layers": [
                "sha256:f353f969c442dec0cd79233d0c0db0ac1458f3811f118587eae5101fb50408a2",
                "sha256:3e08f253054617fd343125239d548ccc7aa33cb3b2296c65ea052afe24dc3452",
                "sha256:683948709559072f615a316b336f2d94f0140a3f5f69eb27c72545cc1321d4aa",
                "sha256:77f03ba195a91b8bf2d444e22bad0837ff955b0a3194dfb85dda18a82fb04ade",
                "sha256:1c07b3a04d96986b4d863a7673e027aafb6ebef2bf4d45498b00ed1be47f4995",
                "sha256:94a95068aa6d3aa9cc9c19edc3b29b1258aa69a57f4f9a5953439dfb441703b5",
                "sha256:bb4b627338bbab2710a680c81daaf67165ae9d14d6bcc66319cb66df6441953a"
            ]
        },
        "Metadata": {
            "LastTagTime": "2021-04-24T20:19:10.8237854Z"
        }
    }
]
```

</details>